### PR TITLE
Searxng theme simple

### PR DESCRIPTION
**Pull request by dalf**
## What does this PR do?

SearXNG brand: 
* Simple theme
* remove hack from https://github.com/searxng/searxng/pull/361
  * https://github.com/searxng/searxng/blob/458575c221abcdbe3c0ffb633dd06f5f6d05bf2e/searx/static/themes/__common__/js/image_layout.js#L161
  * https://github.com/searxng/searxng/blob/458575c221abcdbe3c0ffb633dd06f5f6d05bf2e/searx/static/themes/oscar/src/js/01_init.js#L27

Related to #358

### CODE REVIEW

File: searx/static/themes/simple/src/js/main/searx_keyboard.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
-/*global searx*/
```
Comment (return42) [inline searx/static/themes/simple/src/js/main/searx_keyboard.js:1]: Nit: all files we touch should get a SPDX-License-Identifier, if there is no Copyright a simple tag should be enough:

    /* SPDX-License-Identifier: AGPL-3.0-or-later */

File: searx/static/themes/simple/src/js/main/searx_mapresult.js [hunk 2]
```diff
@@ -1,28 +1,11 @@
-/**
```
Comment (return42) [inline searx/static/themes/simple/src/js/main/searx_mapresult.js:1]: see https://github.com/searxng/searxng/pull/371#discussion_r720340276

File: searx/static/themes/simple/src/js/main/searx_keyboard.js (hunk 1 above)
Comment (dalf) [reply searx/static/themes/simple/src/js/main/searx_keyboard.js:1]: updated

**Review by return42 — APPROVED** [review]
I tested this PR and did not find any issue / approved!

There is only one thing I also wrote in my comments: While we are in the bulk-renaming I think we should add SPDX tags to each file we touch, even if there is no licence remark nor a copyright exists.

**Pull request merged**